package.json

```
@@ -93,6 +93,7 @@
     "metalsmith": "^2.5.0",
     "mocha": "10.2.0",
     "ol-mapbox-style": "^11.0.1",
+    "ol-stac": "^1.0.0-beta.5",
```

```suggestion
    "ol-stac": "^1.0.0-beta.6",
```

To be changed once ol-stac has been rereleased. Makes OL a peer dependency of ol-stac.